attributes/mesos_options.rb

```diff
@@ -380,12 +380,6 @@
       'flag' => false,
       'deprecated' => false
     },
-    'logs_dir' => {
```

Comment:
If this conflicts with the value of node['mesos']['slave']['env']['LOGS'] which is used?